src/oc_erchef/apps/chef_index/src/chef_elasticsearch.erl

```diff
@@ -74,17 +74,21 @@ query_body(#chef_solr_query{
               start = Start,
               rows = Rows}) ->
     chef_index_query:assert_org_id_filter(FilterQuery),
+    TransformQuery = escape_forward_slash(Query),
     jiffy:encode({[{ fields_tag(), <<"_id">>},
         {<<"from">>, Start},
         {<<"size">>, Rows},
         {<<"sort">>, [{[{<<"X_CHEF_id_CHEF_X">>, {[{<<"order">>, <<"asc">>}]}}]}]},
         {<<"query">>, {[
             {<<"bool">>,{[
-                {<<"must">>, {[query_string_query_ejson(Query)]}},
+                {<<"must">>, {[query_string_query_ejson(TransformQuery)]}},
                 {<<"filter">>, {[query_string_query_ejson(FilterQuery)]}}
             ]}}]}
         }]}).
 
+escape_forward_slash(Query) ->
```

Comment:
@piyushawasthi Thanks for working on this and I'm sorry this has gone back and forth so much.  I'm still a bit apprehensive about this bit of the change.  Namely, I'm not sure we need it.  If we exclude it, a query like:

`path:foo/*` 

will fail against elasticsearch.  However, it should be fixable by the user by changing their search to:

`path:foo\/*`

if we include this change, then searching for items with literal `/`s would be very hard.  Perhaps we can chat in person about this.  I know there is at least one user who would have liked this fix a while ago and we've allowed ourselves to delay it for a bit because of the holidays and such.  However, I'd definitely like to get this merged early next week and I think the key is just agreeing on what the behavior should be.

Perhaps @btm or @PrajaktaPurohit can join us to make sure there is some continuity of information.